auth/server/src/main/java/com/seagates3/dao/ldap/LdapStore.java

```diff
@@ -29,11 +29,17 @@
       method.invoke(instance, obj);
     }
     catch (ClassNotFoundException e) {
+      LOGGER.error("Failed to save ");
```

Comment:
@Override public void save(Map<String, Object> dataMap, Object obj,    --> Object is already part of map 
                             String prefix) throws DataAccessException {
    String methodName = "save"; -->define as const string
    String className = "com.seagates3.dao.ldap." + prefix + "LdapStore";   ----->  Repeated in all methods define as const string
    LOGGER.debug("calling method - " + methodName + " of class - " + className);
    try {
      Class < ? > storeClass = Class.forName(className);
      Object instance = storeClass.newInstance();
      Method method = storeClass.getMethod(methodName, Object.class);
      method.invoke(instance, obj);
    }

Reply:
@shalakadharap

Reply:
Fixed